DefensiveItem.h

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
 #include "Item.h"
-#include "Character.h"
+// #include "Character.h"
+struct Character;
```

Comment:
This forward declaration is unnecessary.  `Item.h` is already declaring it
- [ ] revise